Error writing export when using absolute paths on windows

When using the current state of the `develop` branch, trying to execute the JPlag CLI with an absolute path on windows (e.g. like `E:\testSubmissions`) causes JPlag to crash while writing the `options.json` file to the report. Using relative paths works fine though, and I couldn't confirm if this is happening on other platforms than Windows.

JPlag doesn't exit after the error however, and tries to open the report viewer to display the report, which then fails due to the report viewer being unable to read the `options.json` file correctly.
I'm guessing this error got introduced after merging #2437 , since that PR changed how submission paths are exported.

The error is listed below.

```
2025-06-17-05:54:39_432 [INFO] ReportObjectFactory - Start writing report...
2025-06-17-05:54:39_434 [INFO] ReportObjectFactory - Start to export results...
2025-06-17-05:54:56_588 [INFO] OutputFileGenerator - Successfully written the result: results.jplag
2025-06-17-05:54:56_588 [INFO] OutputFileGenerator - View the result using --mode
2025-06-17-05:54:56_589 [INFO] ZipWriter - Summary of all errors:
2025-06-17-05:54:56_583 [ERROR] ZipWriter - Failed to write JSON entry options.json
com.fasterxml.jackson.databind.JsonMappingException: 'other' is different type of Path (through reference chain: de.jplag.options.JPlagOptions["submissionDirectories"]->java.util.Collections$UnmodifiableSet[0])
	at com.fasterxml.jackson.databind.JsonMappingException.wrapWithPath(JsonMappingException.java:400)
        ... 23 more
Caused by: java.lang.IllegalArgumentException: 'other' is different type of Path
	at java.base/sun.nio.fs.WindowsPath.relativize(WindowsPath.java:416)
	at java.base/sun.nio.fs.WindowsPath.relativize(WindowsPath.java:42)
	at de.jplag.reporting.FilePathUtil.forceRelativePath(FilePathUtil.java:37)
	at de.jplag.reporting.jsonfactory.serializer.FileSerializer.serialize(FileSerializer.java:38)
	at de.jplag.reporting.jsonfactory.serializer.FileSerializer.serialize(FileSerializer.java:17)
        ... 21 more

2025-06-17-05:54:56_610 [INFO] JPlagRunner - ReportViewer started on port http://localhost:1996
```

--- a/core/src/main/java/de/jplag/reporting/FilePathUtil.java
+++ b/core/src/main/java/de/jplag/reporting/FilePathUtil.java
@@ -34,7 +34,7 @@ public final class FilePathUtil {
      */
     public static Path forceRelativePath(Path path) {
         if (path.isAbsolute()) {
-            return Path.of("/").relativize(path);
+            return Path.of("./").toAbsolutePath().relativize(path);
         }
         return path;
     }

